### PR TITLE
Improve github issue templates

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Fixes ordering and adds some visual hints:

image

### CODE REVIEW

**Comment by iltar** [comment]
So is "Security" for the Security component? I know the answer, but I'm afraid that someone that thinks "Oh a bug in the security configuration", might follow that link instead.

**Review by xabbuh — APPROVED** [review]

**Review by Oliboy50 — APPROVED** [review]
it would be nice to have links instead of URL in `about`, but I tried both markdown and HTML format and nothing worked

capture d ecran 2018-05-08 a 11 01 02

**Review by ogizanagi — APPROVED** [review]

**Review by lyrixx — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/5_Security_issue.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---
-name: Security Issue
-about: Report security-related errors
```
Comment (fabpot) [inline .github/ISSUE_TEMPLATE/5_Security_issue.md]: I would keep the current description and add the URL after in parenthesis.

**Review by fabpot — CHANGES_REQUESTED** [review]

**Comment by nicolas-grekas** [comment]
@fabpot updated.

Status: needs review

File: .github/ISSUE_TEMPLATE/3_Support_question.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-name: Support Question
-about: Questions about using Symfony and its components
+name: ⛔ Support Question
+about: See https://symfony.com/support for questions about using Symfony and its components.
```
Comment (fabpot) [inline .github/ISSUE_TEMPLATE/3_Support_question.md]: If you add a dot here, you should add it on other `about` section as well... or just remove the ones you added.

**Review by fabpot — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/3_Support_question.md (hunk 2 above)
Comment (nicolas-grekas) [reply .github/ISSUE_TEMPLATE/3_Support_question.md]: dots added

Comment (fabpot) [reply .github/ISSUE_TEMPLATE/3_Support_question.md]: TBH, I would have removed the ones you added

Comment (nicolas-grekas) [reply .github/ISSUE_TEMPLATE/3_Support_question.md]: removed :)

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**